packages/gatsby/src/utils/webpack.config.js

```diff
@@ -469,6 +469,10 @@ module.exports = async (
       runtimeChunk: {
         name: `webpack-runtime`,
       },
+      // use hashes instead of ids for module identifiers
+      // TODO update to deterministic in webpack 5 (hashed is deprecated)
+      // @see https://webpack.js.org/guides/caching/#module-identifiers
+      moduleIds: `hashed`,
```

Comment:
Why is it deprecated though? Is the recommendation to use something else (that is deterministic as well)?

Reply:
The value is "deterministic" not "hashed". It's a new value in webpack 5

Reply:
Oh alrighty